.github/workflows/run-harvest-check.yml

```diff
@@ -29,18 +29,18 @@ jobs:
   #         cf_space: staging
   #         cf_username: ${{secrets.CF_SERVICE_USER}}
   #         cf_password: ${{secrets.CF_SERVICE_AUTH}}
-  # harvest-run-prod:
-  #   name: harvest-run (prod)
-  #   environment: production
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - name: harvest-run
-  #       # pinned to cf7 until --wait is available for run-task on cf8...
-  #       uses: cloud-gov/cg-cli-tools@main
-  #       with:
-  #         command: |
-  #           cf run-task catalog-admin --command 'ckan harvester run' --name 'harvester-check-completed (CI)' -k 2G
-  #         cf_org: gsa-datagov
-  #         cf_space: prod
-  #         cf_username: ${{secrets.CF_SERVICE_USER}}
-  #         cf_password: ${{secrets.CF_SERVICE_AUTH}}
+  harvest-run-prod:
+    name: harvest-run (prod)
+    environment: production
+    runs-on: ubuntu-latest
+    steps:
+      - name: harvest-run
+        # pinned to cf7 until --wait is available for run-task on cf8...
```

Comment:
We aren't actually using this currently (we're using cf8), so this comment isn't valid.